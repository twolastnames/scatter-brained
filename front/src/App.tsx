import { StateDumper } from "./estimation/StateDumper";

function App() {
  return (
    <div data-testid="ScatterBrained">
      <StateDumper />
    </div>
  );
}

export default App;
