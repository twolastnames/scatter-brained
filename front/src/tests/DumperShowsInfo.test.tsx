import App from "../App";

import { render, screen } from "@testing-library/react";

test("can alter application state", () => {
  render(<App />);
  const application = screen.getByTestId("ScatterBrained");

  expect(application).toMatchSnapshot();
});
