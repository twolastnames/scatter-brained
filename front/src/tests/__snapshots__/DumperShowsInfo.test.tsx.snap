// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can alter application state 1`] = `
<div
  data-testid="ScatterBrained"
>
  <span />
</div>
`;
